code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -751,4 +750,31 @@
 		M.adjustStaminaLoss(10)
 	if(prob(30))
 		M << "You should sit down and take a rest..."
-	..()
\ No newline at end of file
+	..()
+
+/datum/reagent/toxin/waste_product
+	name = "Waste Products"
+	id = "waste_products"
+	description = "Natural waste products produced by the metabolism, \
+		only found in great amounts in subjects with renal failure."
+	toxpwr = 0
+	metabolization_rate = 0 // filtered by the kidneys
+	overdose_threshold = 10
+	flags = REAGENT_SILENT_OVERDOSE
+	color = "#8E3975"
+
+/datum/reagent/toxin/waste_product/overdose_process(mob/living/M)
+	// When built up in excess, slowly converts to toxin
+	// but at a rate slower than its normal buildup in humans
+	var/toxlover = FALSE
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(H.dna && H.dna.species && (TOXINLOVER in H.dna.species.specflags))
+			toxlover = TRUE
```

Comment:
Replace with
`toxlover = (H.dna && H.dna.species && (TOXINLOVER in H.dna.species.specflags))`